### ISSUE TITLE
Colorbar tick labels overlap with colorbar title

### ISSUE THREAD

**Issue by jdugge** [plotly/plotly.js#1396]
When using a `colorbar` with `titleside: 'right'` and a small value for `len`, the colorbar tick labels can overlap with the colorbar title:

![image](https://cloud.githubusercontent.com/assets/2399255/23126883/18ab6be0-f779-11e6-8f8e-c5b3060c131a.png)

(Codepen: http://codepen.io/anon/pen/YNoOML)

```
var data = [ {
  z: [[0, 100],
       [75, 0]],
  type: 'heatmap',
  colorbar:{
    len: 0.35,
    title: 'My long label',
    titleside:'right',
    tickvals:[0,50,100],
    tickfont: {color: 'red'}
  }
}]; 

var layout = {};

Plotly.newPlot('myDiv', data, layout);
```

The workaround for avoiding this overlap given in https://github.com/plotly/plotly.js/issues/938#issuecomment-247093233 (using an invisible `tickprefix`) only works for titles that are to the left of the tick labels, unfortunately.

Is there a way to avoid this overlap with titles to the right of the tick labels?

**Comment by hiteshsahu** [comment]
**You need to add <br\> with the title text** 

Add `<br\>` as prefix if you want to add space on top of the title. Add `<br\>` with a `&nbsp;` if you want to add space after the title.
  ```
 colorbar:{
           title: 'Speed(RPM)<br\> &nbsp;',
           titleside:'top',
           titlefont: {color: 'blue'}
         },
```

You can use <br\> trick on other elements of Plotly as well. Hope it help somebody.

**Comment by alexcjohnson** [comment]
closed by #2227
